syntax_and_semantics/methods_and_instance_variables.md

```diff
@@ -44,6 +44,26 @@ peter.age #=> 0
 
 Method names begin with a lowercase letter and, as a convention, only use lowercase letters, underscores and numbers.
 
+## Getters and setters
+
+The Crystal [Standard Library](https://crystal-lang.org/api) provides macros which ease the definition of getter and setter methods:
+
+```crystal
+class Person
+  getter age
+  property name : String
+
+  def initialize(@name)
+    @age = 0
+  end
+end
+
+john = Person.new "John"
```

Comment:
It not really useful to only be able to get the age (not set it), but be able to change the name as we want... I think `name` should be the getter, and `age` the property instead!

Reply:
I think you're right. I started this example intending to replicate the functionality from the previous example with a `#get_older` method but decided it wasn't pertinent to the example.